notebook/static/notebook/js/commandpalette.js

```diff
@@ -151,7 +151,7 @@ define(function(require){
           }
 
           src[group].data.push({
-            display: humanize_action_id(action_id),
+            display: action.cmd || humanize_action_id(action_id),
```

Comment:
Right - if there's a "cmd" string associated, then use that, otherwise, do as before.